src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+## Database Properties
+spring.datasource.url = jdbc:mysql://localhost:3306/users_database?useSSL=false
+spring.datasource.username = root
+spring.datasource.password = Wojna96675
+spring.secondDatasource.url = dbc:mysql://localhost:3306/posts_database?useSSL=false
```

Comment:
Czemu dwie bazy? Jaki to ma sens? Jak będziesz robił joiny pomiędzy tymi dwoma bazami?

Reply:
rev